lib/Driver/Driver.cpp

```diff
@@ -1580,8 +1618,9 @@ void Driver::buildJobs(const ActionList &Actions, const OutputInfo &OI,
   }
 
   for (const Action *A : Actions) {
-    (void)buildJobsForAction(C, cast<JobAction>(A), OI, OFM, TC,
-                             /*TopLevel*/true, JobCache);
+    if (isa<JobAction>(A))
```

Comment:
Style nitpick: instead of doing an `isa` and then a `cast`, use `dyn_cast` and a new variable in the if-condition.